recommend/runtimePolicy.go

```diff
@@ -0,0 +1,150 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2022 Authors of KubeArmor
+
+package recommend
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"os"
+	"strings"
+
+	opb "github.com/accuknox/auto-policy-discovery/src/protobuf/v1/observability"
+
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials/insecure"
+)
+
+var ms MatchSpec
+var filePaths, processPaths SysRule
+
+// CreateRuntimePolicy function generates runtime policy for service account
+func CreateRuntimePolicy(o *Options, img *ImageInfo) error {
+	ms = MatchSpec{}
+	var label string
+
+	for _, labels := range o.Labels {
```

Comment:
I used o.Labels over img.Labels because I was trying to generate something like `app=ubuntu,test=ubuntu` for getting the summary details for a specific pod

Reply:
1) You should convert `img.Labels` to the format accepted `client.GetPodNames()`. Not `o.Labels`. `o.Labels` contains the user given labels in the CLI to filter the deployment. `img.Labels` contains the exact labels of the pod.

2) Since we're passing the `img` object to `summarySearch()`, we can move this label format conversion inside `summarySearch()`